pkg/util/kubernetes/kubelet/kubelet_client.go

```diff
@@ -123,6 +123,17 @@ func (kc *kubeletClient) query(ctx context.Context, path string) ([]byte, int, e
 
 	response, err := kc.client.Do(req)
 	kubeletExpVar.Add(1)
+
+	// telemetry
+	defer func() {
+		code := 0
+		if response != nil {
+			code = response.StatusCode
+		}
+
+		queries.Inc(path, strconv.Itoa(code))
```

Comment:
This will count errors as status code 0. I think we should document that somewhere in the `kubelet/telemetry.go` file to make it clear.